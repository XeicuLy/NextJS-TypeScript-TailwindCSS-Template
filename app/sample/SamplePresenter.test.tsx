import { render } from '@testing-library/react';

import { SamplePresenter } from './SamplePresenter';

describe('app/sample', () => {
  it('renders correctly', () => {
    const { container } = render(<SamplePresenter />);
    expect(container).toMatchSnapshot();
  });
});
