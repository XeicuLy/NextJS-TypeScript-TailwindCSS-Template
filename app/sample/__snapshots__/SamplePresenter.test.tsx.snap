// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app/sample renders correctly 1`] = `
<div>
  <div>
    Sample
  </div>
</div>
`;
